packages/go_router_builder/lib/src/go_router_generator.dart

```diff
@@ -18,7 +18,7 @@ class GoRouterGenerator extends GeneratorForAnnotation<void> {
 
   @override
   TypeChecker get typeChecker => const TypeChecker.fromUrl(
-        'package:go_router/src/route_data.dart#TypedGoRoute',
+        'package:go_router/src/typed_routing.dart#TypedGoRoute',
```

Comment:
@kevmoo Do you know if we change this to `package:go_router/go_router.dart#TypedGoRoute` instead? go_router.dart exports this library.

Reply:
Doesn't seem to work...

Reply:
Note also that changing `go_router_builder` can't be a complete fix for the issue created by 4.2.1; a complete fix must be in `go_router`.

Versions of `go_router_builder` that have this `src` dependency have *already been published* and are in use in the wild, and they depend on `go_router: ^4.0.0`. People using those existing versions can get new versions of `go_router` without getting new versions of `go_router_builder`, so unless you are comfortable breaking those people by violating `semver` you need to revert or otherwise fix `go_router` before publishing it again as 4.x.x.